test/cross-browser-testing/browserstack/browserstack.karma.config.js

```diff
@@ -0,0 +1,73 @@
+const { DEBUG } = process.env;
+
+const files = [
+    '../../lib/mockhttprequest.js',
+    '../../lib/geomock.js',
+    '../../../node_modules/fetch-mock/dist/es5/client-bundle.js',
+    '../../../node_modules/sinon-browser-only/sinon.js',
+    '../../config.js',
+    '../../../dist/mparticle.js',
+    '../../test-bundle.js'
+];
+
+const singleRun = true;
+
+let browserConsoleLogOptions = {
+    terminal: false,
+};
+
+let captureConsole = false;
+
+// Allows console logs to appear when doing npm run test:debug
+if (DEBUG === 'true') {
+    browserConsoleLogOptions = {
+        level: 'log',
+        format: '%b %T: %m',
+        terminal: true,
+    };
+    captureConsole = true;
+}
```

Comment:
I think these can be better organized for clarity.

```suggestion
let captureConsole = false;
let browserConsoleLogOptions = {};

// Allows console logs to appear when doing npm run test:debug
if (DEBUG === 'true') {
    browserConsoleLogOptions = {
        level: 'log',
        format: '%b %T: %m',
        terminal: true,
    };
    captureConsole = true;
} else {
    browserConsoleLogOptions = {
        terminal: false,
    };
}
```